lib/itk/queue/consumer.ex

```diff
@@ -127,8 +127,8 @@ defmodule ITKQueue.Consumer do
 
   defp parse_payload(payload) do
     case use_atom_keys?() do
-      true -> Poison.Parser.parse!(payload, keys: :atoms)
-      _ -> Poison.Parser.parse!(payload)
+      true -> Poison.decode!(payload, keys: :atoms)
```

Comment:
The `Poison.decode!` function just wraps `Poison.Parser.parse!` without doing anything extra.